package.json

```diff
@@ -45,14 +45,16 @@
     "type": "git"
   },
   "scripts": {
-    "build": "npm run lint check-security run-unit-tests enforce-code-coverage",
+    "build": "gulp clean tslint transpile",
```

Comment:
I had trouble getting Travis-CI to execute the original script (that invoked other npm scripts) but can we try adding eslint, enforce-code-coverage, check-security, etc. so that build tools can be updated to do a simple: npm run build?